assets/js/buttons-examples.js

```diff
@@ -159,6 +159,11 @@ $(document).ready(function () {
           label: "Transparent On Hover",
           classes: "transparent-btn",
         },
+        {
```

Comment:
Place this in the correct alphabetical order under Animated

Reply:
> Looks good but is it possible to store the color of the border in a variable? It would be helpful for someone who wants to use it but wants to modify the color.

I think it should be worked on another issue, because border style button are ont the only one this issue, and it will be nice for first timer maybe.

Reply:
Hmm true. I'll create an issue after #395 is resolved. I think we need to organize all variables with good comments in one file so it can be easier to override any color.